tardis/io/decay.py

```diff
@@ -86,4 +86,33 @@ def decay(self, t):
         
         df = IsotopeAbundances.from_materials(decayed_materials)
         df.sort_index(inplace=True)
-        return df 
\ No newline at end of file
+        return df 
+
+    def as_atomic_numbers(self, abundance, t, normalize=True):
+        """
+        Merge decayed Isotope dataframe with abundance passed as parameter 
+
+        Parameters
+        ----------
+
+        t: float or astropy.units.Quantity
+            If float it will be understood as days
+        abundance: pd.DataFrame 
+        normalize : bool
+            If true, resultant dataframe will be normalized
+
+        Returns:
+            : merged abundances
+        """
+        #Drop mass_number coloumn in isotopic_abundance dataframe
+        isotope_abundance = self.decay(t).reset_index(
+            level='mass_number').drop('mass_number', axis=1)
+
```

Comment:
@wkerzendorf `the rest` means normalizing and returning as `model abundance dataframe` format should be moved to other separate function (`as_atomic_numbers`) and only groupby here(`merge_isotopes`)